crates/arkworks/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
I would be in favor of adding a top-level description of what the crate is supposed to contain to guide the next people coming into the repository. In addition to that, we are continuously publishing the documentation and it would be useful for everyone to understand the monorepo structure.